xbmc/network/Network.cpp

```diff
@@ -202,17 +202,19 @@ bool CNetworkBase::IsLocalHost(const std::string& hostname)
 
 CNetworkInterface* CNetworkBase::GetFirstConnectedInterface()
 {
-   std::vector<CNetworkInterface*>& ifaces = GetInterfaceList();
```

Comment:
how does this fix the issue? I don't see this method called by CNetworkBase::WakeOnLan (maybe it is there and I am just blind)

Reply:
Yes you are right it doesn't fix it. I thought `WakeOnLan` is implemented in NetworkInterface but it is implemented in Network and mac address lookup iterates over all interfaces.
Next time I actually should look at the code.